lib/matplotlib/contour.py

```diff
@@ -1475,9 +1475,8 @@ def _check_xyz(self, args, kwargs):
         convert them to 2D using meshgrid.
         """
         x, y = args[:2]
-        kwargs = self.axes._process_unit_info(xdata=x, ydata=y, kwargs=kwargs)
-        x = self.axes.convert_xunits(x)
-        y = self.axes.convert_yunits(y)
+        x, y = self.axes._process_unit_info(
+            [("x", x), ("y", y)], kwargs, convert=True)
```

Comment:
I don't believe you're explicit for this otherwise:
```suggestion
        x, y = self.axes._process_unit_info([("x", x), ("y", y)], kwargs)
```

Reply:
yup